test/viper/invariant.mo

```diff
@@ -4,10 +4,13 @@ actor {
 
   var count = 0 : Int;
 
-  assert claimed and not (-1 == -1) and (-42 == -42) or true;
-  assert count > 0;
+  assert:invariant claimed and not (-1 == -1) and (-42 == -42) or true;
+  assert:invariant count > 0;
+  assert:invariant not claimed implies count == 0;
 
   public shared func claim() : async () {
+      assert:func count >= 0;
+      assert:return count >= 0;
```

Comment:
What do these two lines mean?

Reply:
Is this how preconditions and postconditions being rendered?

Reply:
We don't have dedicated Motoko syntax yet, so they are just Motoko statements.

Reply:
Sounds good. Thanks!